src/Features/Lsif/GeneratorTest/OutputFormatTests.vb

```diff
@@ -64,6 +66,104 @@ Namespace Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.UnitTests
     ""documentSymbolProvider"": false,
     ""foldingRangeProvider"": true,
     ""diagnosticProvider"": false,
+    ""semanticTokensProvider"": {
```

Comment:
Ditto -- it might worth finding a way not trying to hard code this since it'll be unfortunate having to change this any time  we add a new type.